sn/src/node/api/dispatcher.rs

```diff
@@ -186,9 +186,29 @@ impl Dispatcher {
                         .enqueue_and_handle_next_cmd_and_offshoots(cmd, None)
                         .await
                     {
-                        error!("Error sending a cleaning up unused PeerLinks: {:?}", e);
+                        error!(
+                            "Error sending Propose Offline for dysfunctional nodes: {:?}",
+                            e
```

Comment:
You can put the `e` into the format string to cleanup this formatting if you'd like:
```suggestion
                            "Error sending Propose Offline for dysfunctional nodes: {e:?}"
```